server/plugin.go

```diff
@@ -395,6 +413,12 @@ func (p *Plugin) handleChangeAssignment(w http.ResponseWriter, r *http.Request)
 	}
 	r.Body.Close()
```

Comment:
We also want to make sure we always close the HTTP body like we do here

Reply:
@mickmister Added them

Reply:
@hanzei I read the above documentation. From what I can understand, there is no need to close the body here. @mickmister asked me to add them. Please let me know if you think I should remove them or not.

Reply:
IMO they should get removed

Reply:
@hanzei removed it